### DISCUSSION TITLE
Unable to proceed the further case

### DISCUSSION THREAD

**Discussion by Lakshmanan4233** [appium/appium#18840] [Q&A]
I am doing mobile automation (Android) . Some times it is works good . intermettently it isnot working andthrowing error androidx.test.uiautomator.StaleObjectException
Can anyone one you help to understand ? Why the error coming and what the reason behind this.

**Comment by jlipps** [comment]
A StaleObjectException occurs when you try to do something with an element reference and the server no longer has access to the server-side UI element represented by your client-side reference. It means the element you originally found is no longer actually present when you go to interact with it. The solution is usually to try to find the element again.
